### PR TITLE
feat(sap-features-hub): add preinstallation step 1

### PR DESCRIPTION
**Pull request by pauldkn**
ref: MANAGER-15975

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/sap-features-hub`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | feat/MANAGER-15975
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/sap-features-hub/src/components/Form/FormTitle.component.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+};
+
+export const FormTitle: React.FC<FormTitleProps> = ({ title, subtitle }) => {
+  return (
+    <div className="flex flex-col gap-y-8 mb-8">
+      <OdsText preset="heading-2">{title}</OdsText>
+      {subtitle && <OdsText>{subtitle}</OdsText>}
+    </div>
+  );
+};
```
Comment (tibs245) [inline packages/manager/apps/sap-features-hub/src/components/Form/FormTitle.component.tsx:16]:
```suggestion
type FormTitleProps = HTMLAttributes<HTMLDivElement> & {
  title: string;
  subtitle?: string;
};

export const FormTitle: React.FC<FormTitleProps> = ({ title, subtitle, className, ...rest }) => {
  return (
    <div className={`flex flex-col gap-y-8 mb-8 ${className || ''}`} {...rest}>
      <OdsText preset="heading-2">{title}</OdsText>
      {subtitle && <OdsText>{subtitle}</OdsText>}
    </div>
  );
};
```

File: packages/manager/apps/sap-features-hub/src/components/Form/SelectField.component.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  handleChange,
+  options,
+  label,
+  optionLabelKey,
+  optionValueKey,
+  placeholder,
+  error,
+}: SelectFieldProps<T>) => {
+  return (
+    <OdsFormField error={error}>
```
Comment (tibs245) [inline packages/manager/apps/sap-features-hub/src/components/Form/SelectField.component.tsx:41]: Maybe extends all props `OdsFormField` compatible with on this component ?

Comment (pauldkn) [reply packages/manager/apps/sap-features-hub/src/components/Form/SelectField.component.tsx:41]: Not sure of the level of control we'd like this component to have, I think we'll see with our usage if we have to provide `formFieldProps`, `selectProps`, `optionProps`, or `labelProps` in the future, or if it's better to add props step by step depending on our needs

**Review by tibs245 — APPROVED** [review]

File: packages/manager/apps/sap-features-hub/src/components/Form/SelectField.component.tsx (hunk 2 above)
Comment (tibs245) [reply packages/manager/apps/sap-features-hub/src/components/Form/SelectField.component.tsx:41]: 👍

**Pull request merged**